client/resource_manager_client.go

```diff
@@ -158,12 +162,31 @@ func (c *client) DeleteResourceGroup(ctx context.Context, resourceGroupName stri
 	return resp.GetBody(), nil
 }
 
+func (c *client) LoadResourcrGroups(ctx context.Context) ([]*rmpb.ResourceGroup, int64, error) {
+	resp, err := c.Get(ctx, GroupSettingsPathPrefixBytes, WithPrefix())
+	if err != nil {
+		return nil, 0, err
+	}
+	if resp.Header.Error != nil {
+		return nil, resp.Header.Revision, errors.Errorf(resp.Header.Error.Message)
+	}
+	groups := make([]*rmpb.ResourceGroup, 0, len(resp.Kvs))
+	for _, item := range resp.Kvs {
+		group := &rmpb.ResourceGroup{}
+		if err := proto.Unmarshal(item.Value, group); err != nil {
+			continue
+		}
+		groups = append(groups, group)
+	}
+	return groups, resp.Header.Revision, nil
+}
+
 // WatchResourceGroup [just for TEST] watches resource groups changes.
```

Comment:
Can remove `[just for TEST]`

Reply:
This function is still used for testing purposes